opentelemetry-sdk/src/opentelemetry/sdk/_metrics/_internal/aggregation.py

```diff
@@ -176,27 +176,25 @@ def aggregate(self, measurement: Measurement) -> None:
                 self._value = 0
             self._value = self._value + measurement.value
 
-    def collect(self) -> Optional[Sum]:
+    def collect(self, collection_start: int) -> Optional[Sum]:
         """
         Atomically return a point for the current value of the metric and
         reset the aggregation value.
         """
-        now = _time_ns()
-
         if self._instrument_temporality is AggregationTemporality.DELTA:
 
             with self._lock:
                 value = self._value
                 start_time_unix_nano = self._start_time_unix_nano
 
                 self._value = 0
-                self._start_time_unix_nano = now + 1
+                self._start_time_unix_nano = collection_start + 1
```

Comment:
I believe we should remove the `+ 1` based on [this](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/datamodel.md#:~:text=and%20so%20on.-,Delta%20temporality,-means%20that%20successive). Subsequent points should have the same start and end times respectively.

Reply:
Fixed